androidtv/adb_manager/adb_manager_async.py

```diff
@@ -166,11 +166,11 @@ async def connect(self, always_log_errors=True, auth_timeout_s=DEFAULT_AUTH_TIME
                         if not self._signer:
                             self._signer = await self.load_adbkey(self.adbkey)
 
-                        await self._adb.connect(rsa_keys=[self._signer], auth_timeout_s=auth_timeout_s)
+                        await self._adb.connect(rsa_keys=[self._signer], transport_timeout_s=1.0, auth_timeout_s=auth_timeout_s)
```

Comment:
@larena1 are you able to make this change and verify that it fixes the issue? (This is the only line you need to edit.)

Reply:
Couldn't connect to XXX:5555.  TcpTimeoutException: Connecting to XXX:5555 timed out (1.0 seconds)

It will recover correctly now

Reply:
Thanks!